### ISSUE TITLE
[QA] Goerli - Auto dance scene (77,-6): emotes aren't triggered in Alpha while standing on green patches, while it works ok in old explorer

### ISSUE THREAD

**Issue by LucasLioyQA** [decentraland/unity-explorer#1264]
### **Explorer Alpha build version:**

0.3.477-g5ad20358

### **Issue Description:**

 Auto dance scene (77,-6): emotes aren't triggered in Alpha while standing on green patches, while it works ok in old explorer

Old explorer: emotes are properly triggered while standing on green patches: 

https://github.com/decentraland/unity-explorer/assets/146749313/24f4339f-db58-4628-a565-b75da6570cbf

Alpha: emotes aren't triggered while standing on green patches, when they should: 

https://github.com/decentraland/unity-explorer/assets/146749313/0dcc4b4c-7461-4bbb-aa5e-e031daacc69d

### **Steps to reproduce:**

  1. Go to 77, -6 in goerli in alpha
  2. Stand on green patches
  

### **Expected behaviour:**
Emotes should be triggered

### **Current behaviour:**
Emotes aren't triggered

### **Reproduction rate:**
- [ ] < 10%
- [ ]  < = 25%
- [ ]  50%
- [ ]  < = 75%
- [ x]  100%

### **Severity:**

- [ ] Cosmetic
- [ ] Minor
- [x ] Medium
- [ ] High
- [ ] Critical
- [ ] Blocker

### **Priority:**
- [ ] High
- [x ] Medium
- [ ] Low

### **Platforms / OS:**
- [x ] Windows 11
- [ ] Windows 10
- [ ] Mac Sonoma
- [ ] Mac (specify the version on Additional Notes)

**Comment by anicalbano** [comment]
Reopening the issue since the emotes are not being triggered

https://github.com/decentraland/unity-explorer/assets/139157886/38ef49c2-842d-432f-aec7-983b3f90b882

**Comment by nearnshaw** [comment]
Looks like this issue is fixed on the latest commit from today :D

**Comment by lorux0** [comment]
Yep, i can confirm from pr: https://github.com/decentraland/unity-explorer/pull/1369
Be sure to clear your asset bundle cache before re-checking.

https://github.com/decentraland/unity-explorer/assets/56365551/bf5c1f14-0291-49f2-af2f-383d1bae2068
